package.json

```diff
@@ -56,6 +58,9 @@
     "gulp-rename": "^1.2.0",
     "gulp-uglify": "^1.0.2",
     "gulp-watch": "^3.0.0",
+    "qunitjs": "^2.2.1",
```

Comment:
@bumbu when doing a fresh `npm install` without it we get a warning
```
npm WARN qunit-reporter-junit@1.1.1 requires a peer of qunitjs@* but none was installed.
```
Since a recent version of npm, peer-dependencies aren't installed anymore by default.